Informatics II/000/Task 3/Patertuck Solution/Ex00 Pseudocode.txt

```diff
@@ -0,0 +1,19 @@
+Input: An array A[1..n] with n distinct integers, where n ≥ 2. 
+Output: the second largest integer in A
+
+largest_number = very small number
+second_number = very small number
```

Comment:
very small number... 0?

Reply:
if I do it with a very small number it also works with negativ numbers